server/app/views/api/ApiDocsView.java

```diff
@@ -289,40 +302,34 @@ private boolean isAuthenticatedAdmin(Http.Request request) {
   }
 
   private DivTag getNotes(Http.Request request) {
-    String generalDocsLink = "https://docs.civiform.us/it-manual/api.";
     String apiDocsLink = controllers.api.routes.ApiDocsController.index().absoluteURL(request);
 
-    DivTag notesTag = div().withClasses("mt-6");
+    DivTag notesTag = div().withClasses("mt-2");
 
     notesTag.with(
-        text(
-            "The API response preview is a sample of what the API response might look like for a"
-                + " given program. All data is fake. Single-select and multi-select questions have"
-                + " sample answers that are selected from the available responses. General"
-                + " information about using the API is located at "));
-    notesTag.with(
-        a(generalDocsLink)
-            .withHref(generalDocsLink)
-            .withClasses("text-blue-500", "underline")); // create a clickable link
-    notesTag.with(br());
-    notesTag.with(br());
-    notesTag.with(text("You may share this link, "));
-    notesTag.with(
-        a(apiDocsLink)
-            .withHref(apiDocsLink)
-            .withClasses("text-blue-500", "underline")); // create a clickable link
-    notesTag.with(
-        text(
-            ", with anyone in your organization who needs to see API docs, even "
-                + "if they are not a CiviForm Admin or Program Admin."));
-    notesTag.with(br());
-    notesTag.with(br());
-
-    notesTag.with(
-        text(
-            "Note that API docs do not currently support enumerated nor enumerator questions."
-                + " Static content questions are not shown on the API response preview because"
-                + " they do not include answers to questions."));
+        p().withClasses("my-2")
+            .with(
+                text(
+                    "The API response preview is a sample of what the API response might look like"
+                        + " for a given program. All data is fake. Single-select and multi-select"
+                        + " questions have sample answers that are selected from the available"
+                        + " responses.")),
+        p().withClasses("my-2")
+            .with(
+                text("You may share this link, "),
+                a(codeWithStyles(apiDocsLink))
+                    .withHref(apiDocsLink)
+                    .withClasses("text-blue-500", "underline"), // create a clickable link
+                text(
+                    ", with anyone who needs to see API docs. It is accessible without logging in,"
+                        + " so they do not need to be a CiviForm Admin or Program Admin to view"
+                        + " this page.")),
+        p().withClasses("my-2")
+            .with(
+                text(
+                    "Note: These API docs do not currently support enumerated nor enumerator"
+                        + " questions. Static content questions are not shown on the API response"
```

Comment:
Nit & optional: It seems like this note is talking about two unrelated things, enumerator questions and static content questions. It may be slightly clearer to split those out into two paragraphs or bullets instead of combining them into one?

Reply:
Yeah, I removed the bit about static questions cause it seems obvious and is covered by the linked docs.